package.json

```diff
@@ -11,6 +11,7 @@
     "gh-pages": "^2.2.0",
     "react": "^16.13.0",
     "react-dom": "^16.13.0",
+    "react-maskedinput": "^4.0.1",
```

Comment:
What does maskedinput do?

Reply:
It allows for a type of input field where you can type in the date and it will automatically put '/' in the correct spots.